src/components/Lines/Lines.tsx

```diff
@@ -70,7 +70,7 @@ export interface ILinesProps
 			{ x: 'five'  , y0: 4  , y1: 8 , y2: 9 , y3: 8 },
 			{ x: 'six'   , y0: 20 , y1: 8 , y2: 9 , y3: 1 },
 		] */
-	data: Array<{ [key: string]: number } & { x: number | string }>;
+	data: Array<{ [key: string]: number | string }>;
```

Comment:
Noah and I ran into this same issue and here's the solution we came up with: https://github.com/appnexus/lucid/blob/5a6a0f2/src/components/Lines/Lines.tsx#L74-L80